test/cli/commands_test.py

```diff
@@ -1300,22 +1348,3 @@ def test_cli_lint_enabled_progress_bar_multiple_files(
         assert r"\rlint by rules:" in raw_output
         assert r"\rrule L001:" in raw_output
         assert r"\rrule L049:" in raw_output
-
-    def test_cli_lint_disabled_progress_bar_when_verbose_mode(
-        self, mock_disable_progress_bar: MagicMock
-    ) -> None:
-        """Progressbar is disabled when verbose mode is set."""
-        result = invoke_assert_code(
-            ret_code=2,
-            args=[
-                lint,
-                [
-                    "-v" "test/fixtures/linter/passing.sql",
```

Comment:
Note that the test is checking for an error exit code above (`2`). Why? Because of this line. Notice there's no comma between `-v` and the path. This is invalid, so the automatic command-line error checking in `click` causes it to fail before it even gets into the `lint` command. After I fixed this, it seemed like the progress bar *was* enabled, but I don't see the same behavior when I manually test the same scenario. I think some of the class or test-level mocking may be involved.

Reply:
Yeah confirmed on main that `-v` does not disable the progress bar. Found #1873 which fixed the fact it didn't work with progress bars, so agree this test can be removed as should have been removed then.